core/src/test/scala/org/apache/spark/DistributedSuite.scala

```diff
@@ -216,7 +216,7 @@ class DistributedSuite extends SparkFunSuite with Matchers with LocalSparkContex
   test("compute without caching when no partitions fit in memory") {
     val size = 10000
     val conf = new SparkConf()
-      .set("spark.storage.unrollMemoryThreshold", "1024")
+      .set(config.STORAGE_UNROLL_MEMORY_THRESHOLD.key, "1024")
```

Comment:
We might want to use `config.STORAGE_UNROLL_MEMORY_THRESHOLD` directly instead of `.key` if possible.

Reply:
done